apps/server/src/modules/task-card/uc/task-card.uc.ts

```diff
@@ -130,6 +132,31 @@ export class TaskCardUc {
 		return { card, taskWithStatusVo };
 	}
 
+	async setCompletionStateForStudent(studentId: EntityId, taskCardId: EntityId, newState: boolean) {
```

Comment:
newState could be renamed into completed

function could be renamed into toggleCompletionForUser, and we should not use "student" unless we have some code that does something for student only

Reply:
tldr: renaming function to `setCompletionStateForUser` and keep newState variable